Stacktrace if the board is suddenly disconnected

```
null
java.lang.reflect.InvocationTargetException
	at sun.reflect.GeneratedMethodAccessor36.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.execution.process.ProcessHandler$5.invoke(ProcessHandler.java:239)
	at com.sun.proxy.$Proxy19.onTextAvailable(Unknown Source)
	at com.intellij.execution.process.ProcessHandler.notifyTextAvailable(ProcessHandler.java:213)
	at com.intellij.execution.process.BaseOSProcessHandler$SimpleOutputReader.onTextAvailable(BaseOSProcessHandler.java:295)
	at com.intellij.util.io.BaseOutputReader.sendText(BaseOutputReader.java:202)
	at com.intellij.util.io.BaseOutputReader.processInput(BaseOutputReader.java:186)
	at com.intellij.util.io.BaseOutputReader.readAvailableBlocking(BaseOutputReader.java:140)
	at com.intellij.util.io.BaseDataReader.readAvailable(BaseDataReader.java:85)
	at com.intellij.util.io.BaseDataReader.doRun(BaseDataReader.java:163)
	at com.intellij.util.io.BaseDataReader$1$1.run(BaseDataReader.java:66)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:194)
	at com.intellij.util.io.BaseDataReader$1.run(BaseDataReader.java:63)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalStateException: Result is already set
	at com.intellij.util.concurrency.FutureResult.assertNotSet(FutureResult.java:62)
	at com.intellij.util.concurrency.FutureResult.set(FutureResult.java:37)
	at xyz.elmot.clion.openocd.OpenOcdComponent$DownloadFollower.onTextAvailable(OpenOcdComponent.java:192)
	... 20 more
```

--- a/src/xyz/elmot/clion/openocd/OpenOcdComponent.java
+++ b/src/xyz/elmot/clion/openocd/OpenOcdComponent.java
@@ -189,8 +189,10 @@ public class OpenOcdComponent {
         public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType) {
             String text = event.getText().trim();
             if (text.startsWith(ERROR_PREFIX) || text.equals(FLASH_FAIL_TEXT)) {
+                reset();
                 set(STATUS.FLASH_ERROR);
             } else if (text.equals(FLASH_SUCCESS_TEXT)) {
+                reset();
                 set(STATUS.FLASH_SUCCESS);
             }
         }

